tests/storage/__init__.py

```diff
@@ -85,7 +85,9 @@ def add_blob(data):
     observation.add_attribute("region", "1")
 
     response_definition = repository.add_response_definition(
-        name="response_one", indexes_ref=add_blob([3, 5]), ensemble_name=ensemble.name,
+        name="response_one",
+        indexes_ref=add_blob([3, 5, 8, 9]),
+        ensemble_name=ensemble.name,
     )
     active_ref = add_blob([True, False])
```

Comment:
This needs to be extended to match the new size of the observation

Reply:
There are also some further down

Reply:
The size of the observation has not changed, only the size of the responses

Reply:
My bad!